File: Chapter - 1 - Arrays and Strings/1. Is Unique/is_unique.py
def is_unique(str):
	if len(str)>128:
		return False
	char_set=[None]*128
	for i in range(0,len(str)):
		ascii_char=ord(str[i])
		if char_set[ascii_char]:
			return False
		char_set[ascii_char]=True
	return True

print(is_unique("priyapp"))

File: Chapter - 1 - Arrays and Strings/1. Is Unique/is_unique.cpp
#include<iostream>

using namespace std;

bool is_unique(string str) {
    if (str.length()>128) return false;
    bool char_set[128];
    for(int i=0;i<str.length();i++) {
        if(char_set[str[i]]) return false;
        char_set[str[i]]=true;
    }
    return true;
}

int main() {
    string str;
    cout<<"Enter the string you want to check: ";
    cin>>str;
    cout<<is_unique(str);
    return 0;
}
